docs/README.md

```diff
@@ -1,25 +1,25 @@
 # 📚 Dart Frog Site
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+This website is built using [Docusaurus 3](https://docusaurus.io/), a modern static website generator.
 
 ### ⚙️ Installation
 
 ```
-yarn install
+npm install
 ```
 
 ### 🧑‍💻 Local Development
 
 ```
-yarn start
+npm start
```

Comment:
Why are we changing to `npm`? The `docs.yaml` is still `yarn` based. I thin `yarn` claims to be faster than `npm` (I think due to parallel installation instead of sequential).

Reply:
Ah didn't know that. When I did the migration for cli it was npm based. Ill have to go back and re-run things then with yarn.